spec/components/text_sentinel_spec.rb

```diff
@@ -74,6 +74,11 @@
       expect(TextSentinel.new(valid_string.upcase)).not_to be_valid
     end
 
+    it "allows all caps topics when loud posts are allowed" do
+      SiteSetting.stubs(:allow_uppercase_posts).returns(true)
```

Comment:
I don't know if I agree with that one. I'm following the stubbing conventions of what every other test in that file are doing.